install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -771,6 +771,10 @@ data:
   vlan-bpf-bypass: {{ .Values.bpf.vlanBypass | join " " | quote }}
 {{- end }}
 
+{{- if hasKey .Values "enable-k8s-terminating-endpoint" }}
+  enable-k8s-terminating-endpoint: {{ .Values.enableTerminatingEndpoint | quote }}
```

Comment:
I think this one needs a  bit of rework, e.g. above should check `hasKey .Values "enableTerminatingEndpoint"` at min. We should probably also add the entry description to values.yaml. Cc @joestringer

Reply:
Yes, you are right. I forgot to add that file with my changes to the git commit.